Source/Core/DolphinQt2/MainWindow.h

```diff
@@ -105,6 +105,12 @@ class MainWindow final : public QMainWindow
 
   void OnBootGameCubeIPL(DiscIO::Region region);
```

Comment:
Just randomly noticing this: some of the signals are "OnAction", some are just "Action". Could you clean these up and pick one consistent rule?